autoload/ale/handlers/redpen.vim

```diff
@@ -1,5 +1,21 @@
 " Author: rhysd https://rhysd.github.io
 " Description: Redpen, a proofreading tool (http://redpen.cc)
+"
+function! ale#handlers#redpen#ResetOptions() abort
+    call ale#Set('redpen_options', '')
+endfunction
+
+" Reset the options so the tests can test how they are set.
+call ale#handlers#redpen#ResetOptions()
```

Comment:
Remove this code and just call `ale#Set` at the top of the file, which sets a default value. You might want to read the implementation of `ale#Set` in `autoload/ale.vim`.